addons/website/static/src/scss/website.scss

```diff
@@ -1537,6 +1537,12 @@ header {
         @include o-apply-colors('header-sales_four-custom');
         @include o-add-gradient('menu-secondary-gradient');
     }
+} @else if o-website-value('header-template') == 'boxed' {
+    // Use SCSS variable instead of utility class for this header
+    // to ease the maintainability of the code (see .o_mega_menu)
+    .navbar {
+        padding: $navbar-padding-y 0;
+    }
```

Comment:
I'm not sure to understand why this is necessary 🤔 

The navbar already defines by default this variable for vertical paddings : (https://github.com/odoo/odoo/blob/master/addons/web/static/lib/bootstrap/scss/_navbar.scss#L9)

And since you handle the side paddings with an utility class, I think you can remove this part. Or did I missed something ?

Reply:
Ah indeed, I get your point 😅 

So currently, without any change, the padding of navbar is handled by a `py-2` utility class.

To space the mega-menu accordingly, first first that comes to my mind is to reuse the same value, but I can't define `$spacer * .5` because as soon as we'd change this `py-2` the spacing of the megamenu would be different.

----------

But you're right, I think we can just remove this part and keep the megamenu CSS only

Reply:
> But you're right, I think we can just remove this part and keep the megamenu CSS only

I think you can still remove the `py-2` as you did [here](https://github.com/odoo/odoo/pull/171050/files#diff-e9f9ae4b3f4edaeb205bead270f193670b3a0b41cd97482225dd9f7137a12f9dL1460) since the default variable (`$navbar-padding-y`) has the same value 👍 
Tbh, I'm not realy sure why we forced this vertical padding with an utility classe 🤔